### PR TITLE
IGP-24

### PR DESCRIPTION
**Pull request by thrilok209**

### CODE REVIEW

File: contracts/payloads/IGP24/PayloadIGP24.sol [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+            LIQUIDITY.updateUserSupplyConfigs(configs_);
+        }
+
+        // Set user borrow config for the vault on Liquidity Layer.
+        {
+            AdminModuleStructs.UserBorrowConfig[]
+                memory configs_ = new AdminModuleStructs.UserBorrowConfig[](1);
+
+            configs_[0] = AdminModuleStructs.UserBorrowConfig({
+                user: address(supplyToken),
```
Comment (bergben) [inline contracts/payloads/IGP24/PayloadIGP24.sol:443]:
```suggestion
                user: vault,
```

File: contracts/payloads/IGP24/PayloadIGP24.sol [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+                token: borrowToken,
+                mode: 1,
+                expandPercent: 0,
+                expandDuration: 1,
+                baseDebtCeiling: 10 * 1e6,
```
Comment (bergben) [inline contracts/payloads/IGP24/PayloadIGP24.sol:448]: can remove the 1e6 here even for both baseDebtCeiling and maxDebtCeiling. 

I just picked 10 and 100 back then to have some distance between the two values and they can not be 0. Theoretically we could also set 1 and 2 as the absolute lowest possible values, but I thought 10 and 100 is safer. Goal is just to have them very very low close to 0. 10 and 100 USD would also be low enough but just wanted to mention

Comment (thrilok209) [reply contracts/payloads/IGP24/PayloadIGP24.sol:448]: You mean 10 and 100 in wei?

Comment (bergben) [reply contracts/payloads/IGP24/PayloadIGP24.sol:448]: yes

File: contracts/payloads/IGP24/PayloadIGP24.sol [hunk 3]
```diff
@@ -0,0 +1,586 @@
...
+        0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+    address public constant wstETH_ADDRESS =
+        0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0;
+    address public constant weETH_ADDRESS =
+        0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee;
+
+    address public constant sUSDe_ADDRESS =
+        0x9D39A5DE30e57443BfF2A8307A4256c8797A3497;
+    address public constant USDC_ADDRESS = 
+        0xdAC17F958D2ee523a2206206994597C13D831ec7;
```
Comment (bergben) [inline contracts/payloads/IGP24/PayloadIGP24.sol:330]:
```suggestion
        0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
```

File: contracts/payloads/IGP24/PayloadIGP24.sol (hunk 2 above)
Comment (thrilok209) [reply contracts/payloads/IGP24/PayloadIGP24.sol]: Done

**Review by bergben — APPROVED** [review]

**Pull request merged**